### PR TITLE
Minor changes to README

### PR DESCRIPTION
**Pull request by bpradipt**

### CODE REVIEW

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -16,12 +16,14 @@ This project leverages the helpernode [ansible playbook](https://github.com/RedH
...
 	
-The [main](https://github.com/ocp-power-automation/ocp4-upi-powervs/tree/master) branch must be used with latest OCP pre-release versions only. Ensure you have the correct stable branch according the version of RedHat OpenShift you want to install:
-	
-	OCP 4.5: git clone --single-branch --branch release-4.5 https://github.com/ocp-power-automation/ocp4-upi-powervs.git
-	OCP 4.6: git clone --single-branch --branch release-4.6 https://github.com/ocp-power-automation/ocp4-upi-powervs.git
-	OCP 4.7: git clone --single-branch --branch release-4.7 https://github.com/ocp-power-automation/ocp4-upi-powervs.git
+The [main](https://github.com/ocp-power-automation/ocp4-upi-powervs/tree/master) branch must be used with latest OCP pre-release versions only. 
+Ensure you checkout the correct `release-VERSION` branch depending on the version (4.5, 4.6 ...) of RedHat OpenShift you want to install:
 	
+For example to use OCP 4.7 perform the following
```
Comment (mkumatag) [inline README.md:22]: I know it is going to outdated soon, but wondering if we can point atleast to the latest available version which is 4.8

**Review by mkumatag — CHANGES_REQUESTED** [review]
minor comment

File: README.md (hunk 1 above)
Comment (bpradipt) [reply README.md]: Done

**Review by mkumatag — APPROVED** [review]
/lgtm

**Review by Prajyot-Parab — APPROVED** [review]
/approve

**Pull request merged**